libs/community/langchain_community/llms/anyscale.py

```diff
@@ -84,30 +89,58 @@ def send_query(llm, text):
     """
 
     """Key word arguments to pass to the model."""
-    anyscale_api_base: Optional[str] = None
-    anyscale_api_key: Optional[SecretStr] = None
+    anyscale_api_base: str = Field(default=DEFAULT_BASE_URL)
+    anyscale_api_key: SecretStr = Field(default=None)
+    model_name: str = Field(default=DEFAULT_MODEL)
 
     prefix_messages: List = Field(default_factory=list)
 
-    @classmethod
-    def is_lc_serializable(cls) -> bool:
-        return False
+    def __new__(cls, **data: Any) -> BaseOpenAI:  # type: ignore
+        """Initialize the OpenAI object."""
+        model_name = data.get("model_name", "")
+        if model_name not in COMPLETION_MODELS:
+            return AnyscaleChat(**data)
```

Comment:
what is this case for? Should we just throw an error and tell users to use `ChatAnyscale` here instead?

Reply:
Hi Erick, just got back from vacation. Now I have a second thought on it. Yes. I think i should make this client simpler by only accept completion models, and remind ppl to use chat_model client for chat models.
Let's rework and test out your changes as well